react-with-graphql-tutorial.md

```diff
@@ -1522,11 +1525,9 @@ const GET_ISSUES_OF_REPOSITORY = `
 
-Furthermore, the argument `last` has been changed to `first` for the `issues` list field. That's because there wouldn't be another page of items when you would already fetch the last items in the initial request. Thus, you have to start with the first items of the list in order to fetch more and more items until you reach the end of the list eventually.
-
-That's it for fetching the next page of a paginated list with GraphQL in React. One important step is missing though. Nobody updates the local state of the App component yet regarding the new page of issues. Thus, there are still only the issues from the initial request. In order to have this functionality, you need to implement it on your own. Somehow you have to merge the old page(s) of issues with the new page of issues in the local state of the App component. Furthermore, you have to keep the organization and repository information in the deeply nested state object intact. So where would you do this maneuver in the first place? The perfect time for doing it is when the promise for the query resolves. You already extracted it as a function outside of the App component. Thus you can use this place to handle the incoming result and return a result with your own structure and information. Keep in mind, that the incoming result can be an initial request when the App component mounts for the first time or a request to fetch more issues when the "More" button for the pagination feature is clicked.
+That's it for fetching the next page of a paginated list with GraphQL in React, except one final step.  Nothing updates the local state of the App component about a page of issues yet, so there are still only the issues from the initial request. You want to merge the old pages of issues with the new page of issues in the local state of the App component, while keeping the organization and repository information in the deeply nested state object intact. The perfect time for doing this is when the promise for the query resolves. You already extracted it as a function outside of the App component, so you can use this place to handle the incoming result and return a result with your own structure and information. Keep in mind that the incoming result can be an initial request when the App component mounts for the first time, or after a request to fetch more issues happens, such as when the `More` button is clicked.
```

Comment:
"as when the `More` button is clicked" to "as when the "More" button is clicked"